src/views/Fridge.vue

```diff
@@ -48,27 +55,31 @@
 </template>
 
 <script lang="ts">
+import router from "@/router";
 import { Component, Vue } from "vue-property-decorator";
+import { CurrentFamily } from "@/types";
 import Firestore from "@/utils/Firestore";
-import Product from "@/types/Product";
 import NavigationMenu from "@/components/NavigationMenu.vue";
-import VHeader from "@/components/VHeader.vue";
+import Product from "@/types/Product";
 import SearchInput from "@/components/SearchInput.vue";
-import router from "@/router";
-import { CurrentFamily } from "@/types";
+import VAlert from "@/components/VAlert.vue";
+import VHeader from "@/components/VHeader.vue";
 
 @Component({
   components: {
-    SearchInput,
     NavigationMenu,
+    SearchInput,
+    VAlert,
     VHeader
   }
 })
 export default class Fridge extends Vue {
+  alertMessage = "";
```

Comment:
Instead of adding state to each component that wants to use alerts, could we implement a mixin that each component could extend? Something similar to [this](https://class-component.vuejs.org/guide/extend-and-mixins.html#mixins)